schemas/witnet/witnet.proto

```diff
@@ -62,10 +62,7 @@ message Block {
         Hash hash_merkle_root = 3;
     }
     message LeadershipProof {
```

Comment:
As I mentioned in #571, we will be redefining `LeadershipProof` as soon as we implement the VRF.